common/ini-file.h

```diff
@@ -94,40 +97,40 @@ class INIFile {
-	 * For that, it must only consist of letters, numbers, dashes and
-	 * underscores. In particular, white space and "#", "=", "[", "]"
-	 * are not valid!
+	 * For that, it must only consist of letters, numbers, dashes, and
+	 * underscores. In particular, whitespace and "#", "=", "[", "]"
+	 * are not valid.
 	 */
 	bool isValidName(const String &name) const;
 
-	/** Reset everything stored in this ini file. */
+	/** Reset everything stored in this INI file. */
 	void	clear();
 
-	bool	loadFromFile(const String &filename);
-	bool	loadFromSaveFile(const String &filename);
-	bool	loadFromStream(SeekableReadStream &stream);
-	bool	saveToFile(const String &filename);
-	bool	saveToSaveFile(const String &filename);
-	bool	saveToStream(WriteStream &stream);
+	bool	loadFromFile(const String &filename); /*!< Load configuration from a file. */
+	bool	loadFromSaveFile(const String &filename); /*!< Load configuration from a save file. */
+	bool	loadFromStream(SeekableReadStream &stream); /*!< Load configuration from a @ref SeekableReadStream. */
+	bool	saveToFile(const String &filename); /*!< Save the current configuration to a file. */
+	bool	saveToSaveFile(const String &filename); /*!< Save the current configuration to a save file. */
+	bool	saveToStream(WriteStream &stream); /*!< Save the current configuration to a @ref WriteStream. */
 
-	bool	hasSection(const String &section) const;
-	void	addSection(const String &section);
-	void	removeSection(const String &section);
-	void	renameSection(const String &oldName, const String &newName);
+	bool	hasSection(const String &section) const; /*!< Check whether the INI file has a section with the specified name. */
+	void	addSection(const String &section); /*!< Add a section with the specified name to the INI file. */
+	void	removeSection(const String &section); /*!< Remove the @p section from the INI file. */
+	void	renameSection(const String &oldName, const String &newName); /*!< Rename the INI file from @p oldName to @p newName. */
 
-	void	setDefaultSectionName(const String &name); ///< sets initial section name for section-less ini files
+	void	setDefaultSectionName(const String &name); /*!< Set initial section name for section-less INI files. */
 
-	bool	hasKey(const String &key, const String &section) const;
-	bool	getKey(const String &key, const String &section, String &value) const;
-	void	setKey(const String &key, const String &section, const String &value);
-	void	removeKey(const String &key, const String &section);
+	bool	hasKey(const String &key, const String &section) const; /*!< Check whether the @p section has a @p key. */
+	bool	getKey(const String &key, const String &section, String &value) const; /*!< Get the @p value of a @p key in a @p section. */
+	void	setKey(const String &key, const String &section, const String &value); /*!< Assign a @p value to a @p key in a @p section. */
+	void	removeKey(const String &key, const String &section); /*!< Remove a @p key from this @p section. */
```

Comment:
What does removeKey do?
a) Removes the key completely from the section/file.
b) Removes the current value from the key.